content/participant/Jatinkumar.md

```diff
@@ -1,5 +1 @@
----
-name: Jatin Kumar
```

Comment:
Why is this file being edited @prabaljainn ?

Reply:
hey, you got me wrong, actually i Duplicated Jatin Kumar's file and re edited it @CodingSpecies

Reply:
@prabaljainn it shows here that is has been completely deleted though, and there is only one line

Reply:
Did you try to just copy it and then use it in yours, your fileis ok, but there's only one line in the other one?

Reply:
Sorry if i have done anything wrong how to undo this?